src/about-us.md

```diff
@@ -5,22 +5,52 @@ author: elina
+
+ <div class="iframe-container">
   <iframe width="560" height="315" src="https://www.youtube.com/embed/mYzMl_HnEZU" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 </div>
 
-OpenOakland is a non-profit civic innovation organization that brings together coders, designers, data geeks, journalists, and city staff to collaborate on solutions to improve the lives of Oaklanders. We are part of Code for America’s Brigade program and hold frequent events for community, local government and tech folks to work together. Our focus is on both community technology and open
-government projects that we support through community partnerships and engaged volunteers.
-
-Check out our current [projects]({{ site.baseurl }}/projects/). Then get involved by coming to our next virtual [Meetup](https://www.meetup.com/OpenOakland/), every Tuesday, 6:30–9 pm. We look forward to meeting you!
 
-Our organization is led by an elected steering committee and [bylaws](https://docs.google.com/document/d/1QR-fr1WnmXkZoVNmWnZ9drzfmaZoPkodEOx-PkExt94/). We are fiscally sponsored by Code for America, a 501(c)3 nonprofit.
+## Current leadership team
+The leadership team is elected by membership annually.
 
-## Steering Committee
 {% include steering-committee.html %}
 
-A member from each [active project]({{ site.baseurl }}/projects/) team should also attend the Steering Committee meeting each month.
-It doesn't need to always be the same person, but it should be part of the core team. All OpenOakland members are welcome to sit in on leadership meetings. Responsibilities for each steering committee role is defined in the
-[OpenOakland Steering Committee Bylaws](https://docs.google.com/document/d/1QR-fr1WnmXkZoVNmWnZ9drzfmaZoPkodEOx-PkExt94/).
 
-To contact the steering committee, email <a target="_blank" href="mailto:{{ site.email }}">{{ site.email }}</a>.
+
+### Governance and decision-making
+
+We're governed by a set of [bylaws](https://docs.google.com/document/d/1QR-fr1WnmXkZoVNmWnZ9drzfmaZoPkodEOx-PkExt94/) that outline leadership roles, project requirements, decision-making processes, and additional brigade policies.
+
+Formal decision-making is done by the Steering Committee, which meets monthly. Each elected officer has one vote, as does each [active project](/projects/) team. Projects are expected to send a representative to the monthly Steering Committee meetings to exercise their vote. It doesn’t need to be the same person each month, but they should be part of the core team. If a project team doesn't send a rep, they're considered to be abstaining from any vote that month. **All OpenOakland members are welcome to sit in on these meetings.**
+
+![Organization chart depicting the Steering Committee, consisting of elected leadership plus one project representative per project.](/src/assets/images/OpenOakland-governance.png)
```

Comment:
When I ran the project locally, I observed same the error message as the build. It appears that omitting `/src` from the path helps Jekyll find the image correctly. I guess it's different for the .md files versus the .html files which do include `src` in their paths.

```suggestion
![Organization chart depicting the Steering Committee, consisting of elected leadership plus one project representative per project.](/assets/images/OpenOakland-governance.png)
```